src/assets/style.css

```diff
@@ -143,7 +169,22 @@ section#works > .cards > .specialCard {
   row-gap: 12px;
 }
 
-section#works > .cards > .specialCard > h3 {
+section#works > .cards > .otherCards {
+  display: flex;
+  flex-direction: column;
+  row-gap: 24px;
+}
+
+section#about > .hardSkills > ul > li:first-child > h3 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+section#works > .cards > .specialCard > .description > h3 {
```

Comment:
- [OPTIONAL] Good job implementing the changes suggested by previous reviewer, but I think that there is one thing you may need to work on which is the alignment of the feature project. the picture should be aligned left and takes 60% width and the description takes 40% of width and aligned left. I see that requirement works at `1345px` width and above but at lower widths the featured project gets aligned in center. I will highly recommend fixing that.
your design
![qwe](https://user-images.githubusercontent.com/96683843/215151106-9be85b84-b288-4830-9c0b-18ba13301f62.jpg)

figma design
![qwer](https://user-images.githubusercontent.com/96683843/215151129-e5375898-78d5-49d3-bdf6-64290d45747c.jpg)